lib/multiSigTask.ts

```diff
@@ -0,0 +1,385 @@
+      } else if (taskArgs["revokeConfirmation"] !== undefined) {
+        console.log("revoking Confirmation");
+        await hre.run("multiSig:revokeConfirmation", {
+          contract: multiSigContract,
+          proposalId: taskArgs["revokeConfirmation"],
+          signer: signer,
+        });
+      } else if (taskArgs["executeProposal"] !== undefined) {
+        console.log("executing proposal");
+        await hre.run("multiSig:executeProposal", {
+          contract: multiSigContract,
+          proposalId: taskArgs["executeProposal"],
+          signer: signer,
+        });
+      } else if (taskArgs["getProposal"] !== undefined) {
+        console.log("getProposal");
+        await hre.run("multiSig:getProposal", {
+          contract: multiSigContract,
+          proposalId: taskArgs["getProposal"],
+        });
+      } else if (taskArgs["getConfirmations"] !== undefined) {
+        console.log("getConfirmations", taskArgs["getConfirmations"]);
+        await hre.run("multiSig:getConfirmations", {
+          contract: multiSigContract,
+          proposalId: taskArgs["getConfirmations"],
+        });
+      } else if (taskArgs["getTimestamp"] !== undefined) {
+        console.log("getTimestamp");
+        await hre.run("multiSig:getTimestamp", {
+          contract: multiSigContract,
+          proposalId: taskArgs["getTimestamp"],
+        });
+      } else if (taskArgs["isScheduled"] !== undefined) {
+        console.log("isScheduled");
+        await hre.run("multiSig:isScheduled", {
+          contract: multiSigContract,
+          proposalId: taskArgs["isScheduled"],
+        });
+      } else if (taskArgs["isProposalTimelockReached"] !== undefined) {
+        console.log("isProposalTimelockReached");
+        await hre.run("multiSig:isProposalTimelockReached", {
+          contract: multiSigContract,
+          proposalId: taskArgs["isProposalTimelockReached"],
+        });
+      } else if (taskArgs["isFullyConfirmed"] !== undefined) {
+        console.log("isFullyConfirmed");
+        await hre.run("multiSig:isFullyConfirmed", {
+          contract: multiSigContract,
+          proposalId: taskArgs["isFullyConfirmed"],
+          signer: signer,
+        });
+      } else if (taskArgs["isOwner"] !== undefined) {
+        console.log("isOwner");
+        await hre.run("multiSig:isOwner", {
+          contract: multiSigContract,
+          address: taskArgs["isOwner"],
+        });
+      } else if (taskArgs["isConfirmedBy"] !== undefined) {
+        console.log("isConfirmedBy");
+        await hre.run("multiSig:isConfirmedBy", {
+          contract: multiSigContract,
+          proposalId: taskArgs["isConfirmedBy"]["proposalId"],
+          address: taskArgs["isConfirmedBy"]["address"],
+        });
+      } else {
+        if (!taskArgs["getOwners"]) throw new Error("No valid param was given.");
+      }
+    } catch (error) {
+      console.log(error);
+    }
+  });
+
+/// Get owners
+subtask(MULTISIG_GET_OWNERS)
```

Comment:
They were broken down into subtask to avoid cluttering `hardhat --help` messages with all the different functions of the multiSig contract. Since this is not a concern for us, I converted them into individual tasks.